pystiche_papers/johnson_alahi_li_2016/_loss.py

```diff
@@ -54,6 +54,27 @@ def content_loss(
     layer: str = "relu2_2",
     score_weight: Optional[float] = None,
 ) -> ops.FeatureReconstructionOperator:
+    r"""Content_loss from :cite:`JAL2016`.
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper.
+        style: Optional style for selecting the optimization parameters for the
+            replication of the given pre-trained models from the original repository.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            ``omitted``, the default
+            :func:`~pystiche_papers.johnson_alahi_li_2016.multi_layer_encoder` from the
+            paper is used.
+        layer: Layer from which the encodings of the ``multi_layer_encoder`` should be
+            taken. Defaults to "relu2_2".
+        score_weight: Score weight of the operator. If ``omitted``, the score_weight is
+            determined with respect to ``style`` and ``instance_norm``. For details see
+            FIXME
```

Comment:
This should link the table in https://github.com/pmeier/pystiche_papers/blob/master/docs/source/api/pystiche_papers.johnson_alahi_li_2016.rst